instrumentation/aws_lambda/lib/opentelemetry/instrumentation/aws_lambda/version.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module AwsLambda
+      VERSION = '0.1.0'
```

Comment:
@arielvalentin  - I think I got this wrong with the helper gems. Does toys expect `0.0.0`? (sorry about all the churn on this, @xuan-cao-swi)